util/dvsim/NcLauncher.py

```diff
@@ -150,14 +157,51 @@ def poll(self):
         """
 
         assert self.process is not None
-        elapsed_time = datetime.datetime.now() - self.start_time
-        job_runtime_secs = elapsed_time.total_seconds()
         if self.process.poll() is None:
-            timeout_mins = self.deploy.get_timeout_mins()
-            if timeout_mins is not None and not self.deploy.gui:
-                if job_runtime_secs > (timeout_mins * 60):
+            run_timeout_mins = self.deploy.get_timeout_mins()
+            wait_timeout_mins = 180  # max wait time in job queue / license
+            file_size_thresh_bytes = 5000  # log file size threshold
```

Comment:
Probably worth leaving a note here to explain what's going on? (I'd probably also round it up to a multiple of 1024, but maybe that's just me :-D)

Reply:
i have added a one line comment . and changed the threshold to 5120 / 5K

Reply:
Oh, sorry: I was imagining something that gives the reader a bit of a hint about why this is being done. Would something like this work? "We consider the job to have started once its log file contains something. file_size_thresh_bytes is a threshold: once the log file is bigger than this many bytes, the job must have started."

Reply:
done. updated comment

Reply:
I'm really sorry to be annoying, but this probably needs to move down a line so that it's describing `file_size_thresh_bytes` instead of `wait_timeout_mins`.

Reply:
done